src/cloud/internal/cloudservice.cpp

```diff
@@ -78,9 +79,12 @@ void CloudService::init()
     m_oauth2->setAuthorizationUrl(configuration()->authorizationUrl());
     m_oauth2->setAccessTokenUrl(configuration()->accessTokenUrl());
     m_oauth2->setModifyParametersFunction([](QAbstractOAuth::Stage stage, QVariantMap* parameters) {
```

Comment:
It looks like there will now be a compiler warning because the `stage` parameter is unused. Simple solution:
```suggestion
    m_oauth2->setModifyParametersFunction([](QAbstractOAuth::Stage, QVariantMap* parameters) {
```